NICE.Identity.Test/UnitTests/Authorisation.WebAPI/Services/UserServiceTests.cs

```diff
@@ -698,6 +698,51 @@ public async Task TestPendingRegistrationDeletion()
 	        context.Users.Single().NameIdentifier.ShouldBe(user1NameIdentifier);
         }
 
+        [Fact]
+        public async Task TestInActiveAccounts()
+        {
+            //Arrange
+            var context = GetContext();
+            var userService = new UsersService(context, _logger.Object, _providerManagementService.Object, null);
+            const string user1NameIdentifier = "auth|user1";
+            const string user2NameIdentifier = "auth|user2";
+
+            userService.CreateUser(new ApiModels.User
+            {
+                NameIdentifier = user1NameIdentifier,
+                FirstName = "FirstName1",
+                LastName = "LastName1",
+                EmailAddress = "user1@example.com",
+                HasVerifiedEmailAddress = true
+            });
+            userService.CreateUser(new ApiModels.User
+            {
+                NameIdentifier = user2NameIdentifier,
+                FirstName = "User to be deleted",
+                LastName = "",
+                EmailAddress = "user2@example.com",
+                HasVerifiedEmailAddress = true
+            });
+            context.Users.Single(u => u.NameIdentifier == user2NameIdentifier).LastLoggedInDate = DateTime.UtcNow.AddYears(-3);

```

Comment:
Do we also need to test for a user that has also logged in recently, not just a user with no LastLoggedInDate value?

Reply:
Good shout, I've changed the first object so it contains a user with the current date we can test to ensure it's not deleted.